app/src/styling/pages/_landing.scss

```diff
@@ -0,0 +1,88 @@
+$body-bg: #FFDA7C;
+$purple: #6A4C93;
+
+*{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+}
+
+body{
```

Comment:
and same with this body styling, it should go right into the _base.scss file. this doesnt seem too important but its to keep a certain level of 'order' in the project development